uqcs/static/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
+    "short_name": "",
```

Comment:
Would be nice to fill these at some point, though far from necessary in the short term.